__tests__/api.test.js

```diff
@@ -251,5 +251,56 @@ describe('app', () => {
             })
           })
         })
+    })
+
+    describe("7. GET /api/reviews/:review_id/comments", () => {
+    test('should return an array of comment objects relating to review_id', () => {
+      return request(app)
+        .get('/api/reviews/3/comments')
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toBeInstanceOf(Array);
+          expect(comments).toHaveLength(3);
+          comments.forEach((comment) => {
+            expect(comment).toEqual(
+              expect.objectContaining({
+                comment_id: expect.any(Number),
+                body: expect.any(String),
+                review_id: 3,
+                author: expect.any(String),
+                votes: expect.any(Number),
+                created_at: expect.any(String)
+              })
+            )
+          })
+        })
+      })
+      test('204; handles incorrect id', () => {
+        return request(app)
+        .get('/api/reviews/50/comments')
+        .expect(204)
+        .then(({body: { msg }}) => {
+          expect(msg).toBe(undefined)
+        })
+      })
+      test('204; handles id with no comments', () => {
+        return request(app)
+        .get('/api/reviews/1/comments')
+        .expect(204)
+        .then(({body: { msg }}) => {
+          expect(msg).toBe(undefined)
+        })

```

Comment:
This should be a 200 as you would still want to return an empty array of comments for a review that does exist. 

You will have to make another query in your model after checking the rowCount/rows. Check whether you can select this article from the database. If you can, return an empty array. If it doesn't exist, return an error.